specificatie/openapi.yaml

```diff
@@ -36,38 +36,38 @@ paths:
       description: "Free query zoeken van adressen met postcode, woonplaats, straatnaam, huisnummer, huisletter, huisnummertoevoeging. Delen van de adressen in het antwoord matchen exact met jouw invoer. Je vindt een adres door de zoekresultaatidentificatie uit het antwoord te gebruiken in get/adressen"
       parameters:
         - $ref: '#/components/parameters/zoek'
-        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/parameters/page'
-        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/parameters/pageSize'
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.3.0/api-specificatie/common.yaml#/components/parameters/page'
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.3.0/api-specificatie/common.yaml#/components/parameters/pageSize'
       responses:
         '200':
           description: "Geslaagd"
           headers:
             api-version:
-              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/headers/api_version'
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.3.0/api-specificatie/common.yaml#/components/headers/api_version'
```

Comment:
I.v.m. https://github.com/VNG-Realisatie/Haal-Centraal-common/issues/76 zijn nu in deze specs op diverse plaatsen de underscores uit de componentnamen verwijderd.
Zou dat bij de componenten:
* api_version
* X_Pagination_Page
* X_Pagination_Limit
niet ook moeten gebeuren?

Reply:
Dat is een overweging, maar ik wil daar deze release niet op laten hangen. Dit betekent namelijk ook een release in common en dat kost ook weer tijd. Zou je een issue in willen schieten om deze voor versie 1.3 te beoordelen ?

Reply:
Ik was je al voor ;-)
Zie https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/415.